.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
can you make comments in json?

Reply:
my expectations of Microsoft following standards are pretty low so I'm not going to be surprised if they allow it

Reply:
They do follow standard but I can move this into a `*.jsonc` file: https://code.visualstudio.com/docs/languages/json#_json-with-comments

Reply:
I'm an idiot, this is enabled by default for configuration files.

Reply:
So actually, they don't follow the spec.
That is OK by me, if they embrace, extend and extinguish json it will not make me cry ;)